src/utils/math.hpp

```diff
@@ -64,4 +67,44 @@ constexpr std::optional<uint64_t> RoundUint64ToMultiple(uint64_t val, uint64_t m
   return (numerator / multiple) * multiple;
 }
 
+/*!
+ * @return The default precision used in the project. Two doubles are considered different if the distance between them
+ * is strictly greater than Epsilon.
+ */
+constexpr double GetEpsilon() noexcept { return 0.0000001; }
```

Comment:
This is hardcoded. Why did you choose `0.0000001`?

Reply:
Why `0.0000001`: To have a starting point.
To be 100% correct we should have 1 precision per unit (ex: Time, Distance, Weight, etc...) we handle in the application and 1 default configurable by end-user:
- If we know which unit we are handling, we should use the corresponding precision.
- If we can't guess which unit we are handling, use the default one.

I felt it was a bit too much for now. I'd suggest we handle this on `T0886: Double comparison should always include an epsilon. I only wanted to introduce the structure so I could use it.`